content/blog/greenlake-platform-infrastructure-monitoring-with-apache-skywalking.md

```diff
@@ -0,0 +1,144 @@
+
+SkyWalking is an open source observability platform used to collect, analyze, aggregate and visualize data from services and cloud native infrastructures. SkyWalking provides an easy way to maintain a clear view of your distributed systems, even across Clouds. It is a modern APM (Application performance Management), specially designed for cloud native, container based distributed systems.
+
+**Monitoring** is tooling or a technical solution that allows teams to watch and understand the state of their systems. Monitoring is based on gathering predefined sets of metrics or logs.
+
+**Observability** is tooling or a technical solution that allows teams to actively debug their system. Observability is based on exploring properties and patterns not defined in advance.
+
+SkyWalking could be used for monitoring and observability. In this blog post we would focus on how we could setup monitoring of infrastructure (Virtual Machine and/or Bare metal) instances using popular APM tool like SkyWalking.
+
+**Pre-requisite**
+
+* An active service subscription to HPE GreenLake Private Cloud Enterprise.
+* VM and/or Bare metal instances are provisioned for the workloads and we are able to login to console.
+* SkyWalking instance has been setup either on premises or in Hyperscalers. Ensure that inbound ingress traffic is allowed on port 11800 for SkyWalking instance.
+
+ **Steps to configure monitoring**
+
+1. Monitoring host metric with node exporter
+
+* Login to the console of machine to be monitored
+* Download and run the node exporter
+
+```
+wget https://github.com/prometheus/node_exporter/releases/download/v1.4.0-rc.0/node_exporter-1.4.0-rc.0.linux-amd64.tar.gz
+
+tar xvfz node_exporter-1.4.0-rc.0.linux-amd64.tar.gz
+
+rm -rf node_exporter-1.4.0-rc.0.linux-amd64.tar.gz
+
+cd node_exporter-1.4.0-rc.0.linux-amd64
+
+cp node_exporter /usr/sbin
+
+cat<<EOF >nodeexporter.service
+
+[Unit]
+Description=Service to start the node_exporter on the node
+[Service]
+Type=Simple
+ExecStart=/usr/sbin/node_exporter
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=%n
+
+[Install]
+WantedBy=multi-user.target
+
+EOF
+
+
+mv nodeexporter.service /etc/systemd/system
+systemctl daemon-reload
+systemctl enable nodexporter.service
+systemctl start nodexporter.service
+```
+
+ 
+
+*  Check the metrics endpoint shows metrics exported from the infrastructure node
+
+```
+curl <http://localhost:9100/metrics>
+```
+
+ 
+
+![](/img/node_metrics.png "Node Metrics")
+
+2. Setup OTel (OpenTelemetry) collector
+
+OTel provides a set of standardized vendor-agnostic SDKs, APIs, and tools for ingesting, transforming, and sending data to an Observability back-end. We can install OTel collector in the same host which exposes metrics or we can create central VM/Bare metal instance to receive telemetry information from several infrastructure instances.
```

Comment:
In the second sentence, you need a couple of commas:

"We can install OTel collector in the same h**ost, whic**h exposes me**trics, or** we can create..."